.gitignore

```diff
@@ -109,6 +109,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+*.pem
```

Comment:
See this thread: https://stackoverflow.com/questions/25436312/gitignore-not-working